Sources/Citadel/SFTP/SFTPFileFlags.swift

```diff
@@ -126,6 +126,11 @@ public struct SFTPFileAttributes: CustomDebugStringConvertible {
 //    let extended_count: UInt32?
     
     public static let none = SFTPFileAttributes()
+    public static let all: SFTPFileAttributes = {
```

Comment:
I'd prefer a more descriptive name since SFTPFileAttributes holds more than readwrite permissions so `.all` could be a little confusing in that case.

Reply:
We should really be implementing this for permissions: http://learn.leighcotnoir.com/wp-content/uploads/2016/08/POSIX-file-permissions.pdf as optionset . Then we don't need this `.all`